Makefile

```diff
@@ -41,8 +41,7 @@ deps-ubuntu:
 	sudo apt install -y \
 		python3 \
 		python3-pip \
-		libimage-exiftool-perl \
-		libxml2-utils
+		libimage-exiftool-perl
```

Comment:
I assume this reference to `libimage-exiftool-perl` will be gone once https://github.com/OCR-D/core/pull/150 is merged?

Reply:
Correct. Without it, the CI builds break, c.f. https://circleci.com/gh/kba/ocrd-core/217 vs https://circleci.com/gh/kba/ocrd-core/219

Reply:
Oh yeah, that's why we have tests! :-)